tasks/storages.yml

```diff
@@ -1,7 +1,15 @@
 ---
 
 - name: storages > Setup storages
-  command: "pvesm add {{ item.id }} -type {{ item.type }} -maxfiles {{ item.maxfiles|default(1) }} -server {{ item.server }} -export {{ item.export }} -content {{ item.content }} -nodes {{ item.nodes}}"
+  command: >
+    pvesm add {{ item.id }}
+    -type {{ item.type }}
+    -maxfiles {{ item.maxfiles|default(1) }}
+    {% if item.server is defined %} -server {{ item.server }} {% endif %}
```

Comment:
Could you replace such conditional structures in a more ansible-recommended way

Before:
    {% if item.server is defined %} -server {{ item.server }} {% endif %}

After
    {{ (' -server ' ~ item.server) if (item.server is defined) else '' }}
Or
After
    {{ (item.server is defined)|ternary(' -server ' ~ item.server, '' }}